web/packages/teleterm/src/ui/services/connectMyComputer/connectMyComputerService.ts

```diff
@@ -83,26 +82,13 @@ export class ConnectMyComputerService {
 
   async waitForNodeToJoin(
```

Comment:
I realized that when the node fails to join the cluster within the timeout, we should include its logs in the output. I'm going to work on implementing this, in the meantime I submitted a PR which already does 90% of the work.

Reply:
This will require being able to get last 10 logs from the agent at any given time. Doable, we'd just need to store the logs in the AgentRunner state.

https://github.com/gravitational/teleport/blob/91847986b249e7bc835b02f28966164954ef5ddb/web/packages/teleterm/src/mainProcess/agentRunner/agentRunner.ts#L33-L39

What's more, while working on this feature I realized that we do not actually save agent logs anywhere. I incorrectly assumed that they're written to disk by default, but they're actually not and are just sent to stderr.

We should be able to create a logger service per agent and then just save it to disk.